src/NServiceBus.Core/Transports/ISubscriptionManager.cs

```diff
@@ -14,6 +14,11 @@ public interface ISubscriptionManager
         /// </summary>
         Task Subscribe(MessageMetadata eventType, ContextBag context);
```

Comment:
This API is no longer used, I'd like to remove this in a separate PR though to keep this PR focused on the SubscribeAll logic.